### PR TITLE
[miio] Add Yeelight W3 RGB

### PR DESCRIPTION
**Pull request by wespen**
Added config options for Yeelight W3 RGB smart bulb.

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/OH-INF/i18n/basic.properties [hunk 1]
```diff
@@ -3758,4 +3759,3 @@ option.zhimi.humidifier.ca4.mode-0 = Auto
 option.zhimi.humidifier.ca4.mode-1 = Silent
 option.zhimi.humidifier.ca4.mode-2 = Normal
 option.zhimi.humidifier.ca4.mode-3 = Maximum
-
```
Comment (lsiepel) [inline bundles/org.openhab.binding.miio/src/main/resources/OH-INF/i18n/basic.properties:3761]: Doesn't the empty line need to stay?

**Review by lsiepel — COMMENTED** [review]
One minor comment,  please also look ath the DCO, otherwise LGTM

**Comment by marcelrv** [comment]
Thanks for the contribution.
Can you please run the 'ReadmeHelper.java' in the src/test/java folder as that will ensure the readme and .properties files are updated in the right way. It most likely also addresses directly the question of @ lsiepel

**Comment by marcelrv** [comment]
@wespen you need any support to get this through?

**Comment by wespen** [comment]
> @wespen you need any support to get this through?

Sorry, missed these comments.
I saw it was added in https://github.com/openhab/openhab-addons/pull/16659

Thx for pushing this through.

**Pull request closed without merging**